### PR TITLE
feat: prepare new components ui (ACC-82)

### PR DESCRIPTION
**Pull request by przemvs**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/react-ui-kit/src/Misc/IconButton.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (thisisamir98) [inline packages/react-ui-kit/src/Misc/IconButton.tsx:3]:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```

File: packages/react-ui-kit/src/Misc/ButtonGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (thisisamir98) [inline packages/react-ui-kit/src/Misc/ButtonGroup.tsx:3]:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```

File: packages/react-ui-kit/src/Misc/ButtonGroup.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Demo:
```
Comment (thisisamir98) [inline packages/react-ui-kit/src/Misc/ButtonGroup.md:1]: license

File: packages/react-ui-kit/package.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
     "@emotion/jest": "11.9.1",
     "@hot-loader/react-dom": "17.0.1",
     "@types/jest": "26.0.24",
-    "@types/react": "18.0.5",
+    "@types/react": "18.0.9",
```
Comment (thisisamir98) [inline packages/react-ui-kit/package.json:22]: isn't there a 18.1.0?

File: packages/react-ui-kit/package.json [hunk 5]
```diff
@@ -35,7 +35,7 @@
   },
   "homepage": "https://wire-react-ui-kit.netlify.app/",
   "peerDependencies": {
-    "@types/react": "^17.0.0",
+    "@types/react": "^18.0.9",
```
Comment (thisisamir98) [inline packages/react-ui-kit/package.json:38]: same here? 18.1.0?

File: packages/react-ui-kit/src/Form/Checkbox.tsx [hunk 6]
```diff
@@ -78,7 +96,7 @@ const filterCheckboxProps = (props: CheckboxProps) => filterProps(props, ['markI
 export const Checkbox: React.FC<CheckboxProps<HTMLInputElement>> = React.forwardRef<
   HTMLInputElement,
   CheckboxProps<HTMLInputElement>
->(({id = Math.random().toString(), children, style, disabled, ...props}, ref) => (
+>(({id = useId(), children, style, disabled, ...props}, ref) => (
```
Comment (atomrc) [inline packages/react-ui-kit/src/Form/Checkbox.tsx:99]: Oh that's cool, I was not aware React had this :ok_hand:

Comment (tlebon) [reply packages/react-ui-kit/src/Form/Checkbox.tsx:99]: nice this caught my eye as well

File: packages/react-ui-kit/src/Misc/IconButton.tsx (hunk 1 above)
Comment (tlebon) [reply packages/react-ui-kit/src/Misc/IconButton.tsx]: this should still be updated?

File: packages/react-ui-kit/src/Misc/ButtonGroup.tsx (hunk 2 above)
Comment (tlebon) [reply packages/react-ui-kit/src/Misc/ButtonGroup.tsx]: this also?

**Review by tlebon — CHANGES_REQUESTED** [review]

**Pull request merged**